### PR TITLE
Add support to upgrade to the latest version

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Adds an option to upgrade to the latest version available
  in the channel when set.
- Fixes linting issues.

### CODE REVIEW

File: snafu/upgrade_openshift_wrapper/trigger_upgrade.py [hunk 1]
```diff
@@ -73,10 +74,19 @@ def _run_upgrade(self):
                 clusterversion.get().attributes.items[0].status.history[0].completionTime,time_format)
             return init_version, self.version, platform, the_time, the_time, (the_time - the_time)
 
+        # Fail if both toimage and to_latest parameters are set
+        if self.toimage and self.latest:
+            logging.error("Looks like both toimage and to_latest parameters are set, please set one of them")
```
Comment (jtaleric) [inline snafu/upgrade_openshift_wrapper/trigger_upgrade.py:79]: Hm, should `toimage` just take president?

Comment (jtaleric) [reply snafu/upgrade_openshift_wrapper/trigger_upgrade.py:79]: Actually.. thinking more, this is good.. just exit(1) here..

File: snafu/upgrade_openshift_wrapper/upgrade_openshift_wrapper.py [hunk 2]
```diff
@@ -33,6 +33,10 @@ def __init__(self, parent_parser):
         parser.add_argument(
             '--toimage',
             help='Provide the target image location')
+        parser.add_argument(
+            '--to_latest',
```
Comment (jtaleric) [inline snafu/upgrade_openshift_wrapper/upgrade_openshift_wrapper.py:37]: why `to_latest` vs `latest` ?

**Review by jtaleric — CHANGES_REQUESTED** [review]
one small nit.

File: snafu/upgrade_openshift_wrapper/upgrade_openshift_wrapper.py (hunk 2 above)
Comment (chaitanyaenr) [reply snafu/upgrade_openshift_wrapper/upgrade_openshift_wrapper.py]: Switched to latest.

**Review by jtaleric — APPROVED** [review]
LGTM

**Comment by mohit-sheth** [comment]
LGTM

**Pull request merged**